thirdparty/intervaltree/IntervalTree.h

```diff
@@ -67,7 +67,7 @@ class IntervalTree {
         center = other.center;
         intervals = other.intervals;
         if (other.left) {
-            left = (intervalTree*) malloc(sizeof(intervalTree));
+            left = new intervalTree();
```

Comment:
not sure we really want to change third party code?

Reply:
Although `new` is used in thos file just a few lines below...

Reply:
You're right; it looks like [upstream](https://github.com/ekg/intervaltree) has fixed this; are there any plans to merge it?

Reply:
I'm not aware of any such plans.

Reply:
I'm not aware of any such plans.

Reply:
Alright, I've reverted that and created a new issue merging upstream at #8808.

Reply:
Ah, I see, even better